benchmarks/wrappers/fission/nodejs/handler.js

```diff
@@ -0,0 +1,31 @@
+const path = require('path'), fs = require('fs');
```

Comment:
For all wrappers, we need to compare them against newest implementation in OpenWhisk/Knative - there have been changes. In particular, storage triggers received few bugfixes.